### ISSUE TITLE
Question - How do I anchor axis range label text to the start?

### ISSUE THREAD

**Issue by smartb-pair** [amcharts/amcharts4#353]
I give up: If I have a date axis range label which has a single date value, how do I anchor the text at the start so that it extends to the right, instead of being in the middle over the top of the line? The "align" and "textAlign" properties have no effect on the position of the text, presumably because they are only relevant if the range has a start and end date.

**Comment by notacouch** [comment]
Thanks for describing the setup in detail.

To align the axis range label in this case you'll want to use the [`horizontalCenter`](https://www.amcharts.com/docs/v4/reference/axislabel/#horizontalCenter_property) property:

> Controls which part of the element to treat as a horizontal center.
>
> The setting will be used when positioning, resizing and rotating the element.

Values of interest are `"left"` and `"middle"` (all the other valid values can be found here: https://www.amcharts.com/docs/v4/reference/horizontalcenter/).

If you set it to `"left"`, the label's left edge will be flush against the range and vice versa (albeit with a default padding of 10px).

However, instead of just setting it directly (`range.label.horizontalCenter = "left";`), we'll follow [the best practice according to documentation](https://www.amcharts.com/docs/v4/concepts/axes/axis-ranges/#Adding_ranged_label), and use adapters to enforce our design decision over any chart rendering calculations::

```javascript
range.label.adapter.add("horizontalCenter", function() {
  return "left";
});
```

Here's a demo with all that thrown together with some extra tidbits found in the comments:
https://codepen.io/team/amcharts/pen/57a2ec42ce2d19d747a48e330cf1f3dc/

Screenshot:
![image](https://user-images.githubusercontent.com/2334143/45582261-b65eff00-b87a-11e8-98f8-aebe5157b8ff.png)

Let us know if this helps.

**Comment by smartb-pair** [comment]
wow thanks @notacouch for the super detailed response! while i'm here i will be lazy and ask another question because i can't be bothered looking through the docs - is it possible to position the label text further up the line, instead of it always being at the bottom? there will be cases when i will have range labels which are very close together and i need the text to not overlap so would be good if i could vary the position at which they appear.

**Comment by notacouch** [comment]
@smartb-pair  you're welcome!

**TL;DR**: `range.label.dy = ##`, where `##`, e.g. `-25`, is a number in pixels. It will shift the label that many pixels up or down (negative moves it up).

Explanation:
In this case, the vertical position of an axis label is determined against its axis, so it's either outside (default) or `inside`. There's not going to be some setting to vertically align it according to the range guide. So all we've got is `dy`.

Threw another pen together real quick:
https://codepen.io/team/amcharts/pen/3546145c9eec9fb06a67ce4fc5be4a27

Screenshot:
![image](https://user-images.githubusercontent.com/2334143/45629986-b53df580-ba65-11e8-8d95-87fb1d3aa544.png)

**Comment by smartb-pair** [comment]
Theeeenks!
